index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+
+    <h1>My People</h1>
+
+
+
+
+    <script id="store-template" type="text/x-handlebars-template">
+        {{#each people}}
+            <div id='results-container'>
+                <div class='result'>
+                    <div class='name'>Name:</div>
+                    <div class='flname' style="padding-left:5em">{{firstName}} {{lastName}}</div>
+                    <div class='l-age'>Age:</div>
+                    <div class='age' style="padding-left:5em" >{{age}}</div>
+                    <div class='l-cell'>cell:</div>
+                    <div class='cell'style="padding-left:5em">{{cell}} </div>
+                    <div class='l-email'>email: </div>
+                    <div class='email'style="padding-left:5em"> {{email}}</div>
+                    <div class='img'><img src="{{photo}}"></div>
+
+                </div>    
+            </div>    
+        {{/each}}
+    </script>
+
+    
+    
+    <div id = "results"></div>
```

Comment:
the script tags should be at the bottom of the HTML file right before the `</body>`
this `<div id = "results"></div>` should be before the script tags